apollo-compiler/src/test/graphql/com/example/unique_type_name/HeroDetailQuery.kt

```diff
@@ -62,6 +62,7 @@ class HeroDetailQuery : Query<HeroDetailQuery.Data, HeroDetailQuery.Data, Operat
 
     companion object {
       private val RESPONSE_FIELDS: Array<ResponseField> = arrayOf(
+          ResponseField.forString("__typename", "__typename", null, false, null),
```

Comment:
Does it make sense that the same line is next to each other here?

Reply:
Do you mean the same line of  `ResponseField.forString("__typename", "__typename", null, false, null)`?  If yes that's correct, first line used by actual field `__typename` and the second one is not involved in parsing response, but it needs to be there to have right field index.

I'm thinking of making changes to remove `RESPONSE_FIELDS ` all together, as index based access to the response field meta looks fragile.